docs/usage.txt

```diff
@@ -62,6 +62,11 @@ Result:
 
         <link rel="preload" href="/static/CACHE/js/d01466eb4fc6.js" as="script" />
 
+.. note::
+
+    This operation only generates the preload HTML link element with the correct filename hash.
+    You must subsequently load the file within a ``compress`` block to use the associated resource.
```

Comment:
Good point.  And I am realizing that we should probably also clarify that it needs the same block name.

Reply:
Hi, what do you mean by "same block name"?

Reply:
django-compressor supports setting a block name that will be prepended to the file name for a compressed resource.  The block name used must be the same between the preload block and the subsequent file block for the file names to match.

Reply:
Ok, I'll try that since I still have the error with offline compression at runtime:
`compressor.exceptions.OfflineGenerationError: You have offline compression enabled but key "d194e833cc7675b07be5a9dee9eea326d443e6f691265734b1b3abeb2b7982be" is missing from offline manifest. You may need to run "python manage.py compress". Here is the original content`

We have identical couples of compress tags (preload + non-preload) in several templates, maybe that's the issue.

Reply:
Hmm... that error may be something different.  Possibly related to #961 ?  The last commit above (e35713a) was a quick and dirty attempt to solve that issue.

Reply:
That's what I thought and why I tried your PR. Maybe It is a corner case with inheritance and block tag inside compress tags.
I just jumped in the project for a technical migration and found the offline compression appealing.
Anyway, I'll stick to the online compression for now, many thanks for your time.